### PR TITLE
Feature one

### PR DESCRIPTION
**Pull request by mesielepush**
Pull request to merge the final section of the weather app to the Development branch. All systems work, linters passes and UI is not ugly.

### CODE REVIEW

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @mesielepush 

Really nice and original UI :smiley_cat: I really love it.
However, although everything is working correctly, one thing is out to the requirement.
Screenshot at 2020-07-05 12:25:08
That screenshot show that every time the user click on the toggle button to change the temperature unit, the application does a request to the API. Which is not good.
The conversion from Fahrenheit to Celsius and vice-versa must be done locally without any request.

Kindly make the changes and submit a new code review request
Thanks

Mael FOSSO

**Comment by mesielepush** [comment]
Doing separate requests for Celcius and Fahrenheit it's a microverse requirement.
image

File: src/js/central.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        tempMax: weatherData.list[0].main.temp_max,
+        tempMin: weatherData.list[0].main.temp_min,
+        feels: weatherData.list[0].main.feels_like,
+        country: weatherData.city.country,
+      };
+
+      return data;
+    } catch (err) {
+      // eslint-disable-next-line no-alert
+      alert('City does not exist');
```
Comment (bellom) [inline src/js/central.js:28]: ### [Optional]
This should be production code, we shouldn't have `alert`, please remove the `alert`.

**Review by bellom — APPROVED** [review]
## Hi @mesielepush :clap: 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**